inventory/hosts.example

```diff
@@ -341,7 +341,10 @@ debug_level=2
 #osm_controller_args={'resource-quota-sync-period': ['10s']}
 
 # Configure api server arguments
-#osm_api_server_args={'max-requests-inflight': ['400']}
+#osm_api_server_args={'max-requests-inflight': ['400'],'experimental-encryption-provider-config':['/etc/origin/master/encryption-config.yaml']}
+
+# Configure openshift encrypting config
+#openshift_encryption_config="{'kind': 'EncryptionConfig','apiVersion': 'v1','resources': [{'resources': ['secrets'],'providers': [{'aescbc': {'keys': [{'name': 'key1','secret': 'xxxxxxxx'}]}},{'identity': {}}]}]}"
```

Comment:
What is the desired format of this data?  Currently, this is a python dictionary serialized into a string.  If you are looking to have json here, need to use double quotes for elements and single quotes to wrap the entire value.